kodilla-advanced-tests/src/main/java/com/kodilla/execution_model/homework/Shop.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.execution_model.homework;
+
+public class Shop {
```

Comment:
public void addOrder(Order order) {
  // do nothing
}

public List<Order> getOrdersFromARangeOfDates(LocalDate from, LocalDate to) {
   return null;
}